File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_advanced_-_exercise/cinema.py
type_projection = str(input())
r = int(input())
c = int(input())

price = 0

if type_projection == "Premiere":
    price = 12.00

elif type_projection == "Normal":
    price = 7.50

elif type_projection == "Discount":
    price = 5.00


print("{:.2f}".format((r * c) * price) + " leva")

File: JS/JavaScript/JS-Basics/Conditional StatementsAdvancedExercise/Cinema.js
function cinema(inputs) {

    let typeTicket = String(inputs[0])
    let liens = Number(inputs[1])
    let columns = Number(inputs[2])

    let premierePrice = 12.00
    let normalPrice = 7.50
    let discount = 5.00

    let hall = liens * columns

    if (typeTicket == 'Premiere') {
        console.log((hall * premierePrice).toFixed(2) + " leva");
    }

    else if (typeTicket == 'Normal'){
        console.log((hall * normalPrice).toFixed(2) + " leva");
    }

    else if (typeTicket == 'Discount'){
         console.log((hall * discount).toFixed(2) + " leva");
    };

}

cinema(["Premiere","10","12"]);

cinema(["Normal","21","13"]);

cinema(["Discount","12","30"]);

